tests/layer_tests/common/utils/common_utils.py

```diff
@@ -48,15 +48,36 @@ def generate_ir(coverage=False, **kwargs):
 
 
 def generate_ir_python_api(coverage=False, **kwargs):
-    from openvino.runtime import serialize
-    from openvino.tools.mo import convert_model
-
+    from openvino.runtime import save_model
+    from openvino.tools import mo, ovc
+    
     out_dir = kwargs['output_dir'] + os.sep + kwargs['model_name'] + ".xml"
-
-    # TODO: Remove usage of legacy params from layer tests and switch to convert_model from tools.ovc
-    ov_model = convert_model(**kwargs)
-    serialize(ov_model, out_dir)
-
+    compress_to_fp16 = getattr(kwargs, 'compress_to_fp16', True)
+
+    import inspect
+    acceptable_mo_arg_names = ['use_new_frontend', 'model_name', 'compress_to_fp16', 'output_dir']  # mo args that do not conflict with ovc
+    ovc_arg_names = inspect.signature(ovc.convert_model).parameters.keys()
+
+    # By default, run in ovc, but fallback to mo if there are args not present in ovc.convert_model.
+    # Also, if there are such arguments as 'model_name', 'out_dir', etc. they do not conflict with ovc, they should be just
+    # removed from kwargs and then ovc.convert_model can be run with updated ovc_kwargs.
+    use_ovc = True
+    
+    ovc_kwargs = kwargs.copy()
+    for key in kwargs.keys():
+        if key not in ovc_arg_names:
+            ovc_kwargs.pop(key)
+        if key not in ovc_arg_names and key not in acceptable_mo_arg_names:
+            use_ovc = False
+            break
+
+    if use_ovc:
+        ov_model = ovc.convert_model(**ovc_kwargs)    
+    else:
+        # TODO: Remove usage of legacy params from layer tests and switch completely to ovc.convert_model
+        ov_model = mo.convert_model(**kwargs)    
+    
+    save_model(ov_model, out_dir, compress_to_fp16)
```

Comment:
Why do we need these changes? Let us do this in separate PR

Reply:
Since marking is called inside `save_model` it was needed to call `save_model` in layer tests so that this fix work. I tried to call `ovc + save_model` instead of `mo + serialize` in cases where it's possible. But since it didn't work well, i reverted it.
Agreed, let's consider using ovc in a separate PR.

Now i always call `mo + save_model`. @popovaan, could you please confirm that always calling `save_model` is ok?

Reply:
Fully reverted common layer test changes. Switching to ovc will be done under CVS-132151

Xfailed my newly added tests since for them to be green `save_model` should be called which impossible to call without common layer test changes. When i run locally FloorDiv layer tests with `save_model` they are green
![image](https://github.com/openvinotoolkit/openvino/assets/5703039/01e6436b-2e8d-42d7-9ac0-efb339cfcb0f)